documents/entityrelationship/model.go

```diff
@@ -215,12 +215,12 @@ func (e *EntityRelationship) CreateProofs(fields []string) (proofs []*proofspb.P
 	return e.CoreDocument.CreateProofs(e.DocumentType(), tree, fields)
 }
 
-// DocumentType returns the entity relationship document type. THIS IS NOT IMPLEMENTED FOR ENTITY RELATIONSHIP.
+// DocumentType returns the entity relationship document type.
 func (*EntityRelationship) DocumentType() string {
 	return documenttypes.EntityRelationshipDocumentTypeUrl
 }
 
-// AddNFT adds NFT to the EntityRelationship.
+// AddNFT adds NFT to the EntityRelationship. THIS IS NOT IMPLEMENTED FOR ENTITY RELATIONSHIP.
```

Comment:
I didn't notice this before, 
Please don't panic on production code specially since this something someone can try by calling nft endpoint, just log and return an error. 
Only places where we call panic is when the node shouldn't be alive afterwards such as when there is a config error.

Reply:
```suggestion
// AddNFT is not implemented for EntityRelationship.
```

Reply:
Panic is still there right?